packages/devtools-launchpad/webpack.config.devtools.js

```diff
@@ -22,9 +22,7 @@ module.exports = (webpackConfig, envConfig) => {
   }
 
   webpackConfig.devtool = false;
-  webpackConfig.recordsPath = path.join(
-    rootDir, "assets/module-manifest.json"
-  );
+  webpackConfig.recordsPath = path.join(os.tmpdir(), "module-manifest.json");
```

Comment:
Okay, I'll revert this for now and we can leave it for #254. Possibly related to linking. The side effects of linking are quite frustrating since they _are_ quite convenient, but then end up breaking various things...